app/src/main/java/com/islaharper/dawnofandroid/domain/model/RemoteKeys.kt

```diff
@@ -0,0 +1,13 @@
+package com.islaharper.dawnofandroid.domain.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "remote_keys")
+data class RemoteKeys(
```

Comment:
What are these remote keys and why are we storing them in the db? Is it just so we know where we have paged to? Can we add some KDoc?